.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: generic
+services:
+  - docker
+
+notifications:
+  email:
+    on_success: change
+    on_failure: always
+env:
+  global:
+    - ROS_REPO=ros
+    - NOT_TEST_INSTALL=true
+    - VERBOSE_TESTS=false
+    - UPSTREAM_WORKSPACE=file
+  matrix:
+    - ROS_DISTRO=kinetic
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+script:
+  - .ci_config/travis.sh
+#  - ./travis.sh  # Enable this when you have a package-local script
```

Comment:
this is not an up-to-date travis config

Reply:
I updated with the `.travis.yml` from https://github.com/mojin-robotics/mojin_flexbe/blob/1f9ad52902c7cd3e8066f956fdb724d9972b5b3a/.travis.yml because it was the latest commit I found

Reply:
check any other repo - even `mojin_flexbe` for the latest `.travis.yml`
this is a good example: https://github.com/mojin-robotics/mojin_drivers/blob/kinetic-devel/.travis.yml